### PR TITLE
Use .NET 4.5's version of ZipArchive - references #729

### PR DESCRIPTION
**Pull request by forki**
This chance has possibly big implications.

Please review it and then I can create an prerelease version for broader testing.

/cc @theimowski @mexx @mavnn @Vidarls

### CODE REVIEW

File: src/Paket.Core/NuGetV2.fs [hunk 1]
```diff
@@ -331,15 +330,8 @@ let ExtractPackage(fileName:string, targetFolder, name, version:SemVerInfo) =
...
-            use zip = ZipFile.Read(fileName)
             Directory.CreateDirectory(targetFolder) |> ignore
-            for e in zip do
-                try
-                    e.Extract(targetFolder, ExtractExistingFileAction.OverwriteSilently)
-                with
-                | :? Ionic.Zip.BadCrcException as exn -> 
-                    traceWarnfn "Bad Crc during unzipping %s in %s %A: %s" e.FileName name version exn.Message 
-                | exn -> failwithf "Error during unzipping %s in %s %A: %s" e.FileName name version exn.Message 
+            ZipFile.ExtractToDirectory(fileName, targetFolder)
```
Comment (theimowski) [inline src/Paket.Core/NuGetV2.fs]: Is overwrite silently the default behavior? And how about Crc exn? Not thrown anymore?

**Comment by theimowski** [comment]
Lgtm

File: src/Paket.Core/NuGetV2.fs (hunk 1 above)
Comment (forki) [inline src/Paket.Core/NuGetV2.fs]: not sure.

**Comment by forki** [comment]
@mavnn most things work, but nuget package explorer tells me about missing manifest in generated package. any ideas?

**Comment by forki** [comment]
I tried to diff the packages, but I don't see any differences. I'm stuck.

**Pull request merged**